src/Utils/Collection.php

```diff
@@ -0,0 +1,7 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Libs\Util;
+
+class Collection extends \ArrayObject {
```

Comment:
What's the purpose of the `Collection` class? Why isn't `FieldCollection` just extending `ArrayObject`?